robot/web/stream/stream.go

```diff
@@ -78,15 +84,19 @@ func (opts *BackoffTuningOptions) getErrorThrottledHandler(logger logging.Logger
 		}
 		lastErrTime = now
 
-		if errors.Is(prevErr, err) {
+		switch {
+		case errors.Is(prevErr, err):
+			errorCount++
+		case prevErr != nil && prevErr.Error() == err.Error():
```

Comment:
Correct -- it is never nil. I think the easiest way to convince oneself that is likely true (without looking up the callers) is that (oldcode) line 89 is logging `err` as an error log.

This lambda is just wrapping error handling code to avoid repeating error logs in tight loops (like reading video data).